clients/ts/signalr/package.json

```diff
@@ -38,10 +38,12 @@
   "devDependencies": {
     "es6-promise": "^4.2.2",
     "@types/node": "^10.9.4",
-    "@types/eventsource": "^1.0.2"
+    "@types/eventsource": "^1.0.2",
+    "@types/request": "^2.47.1"
```

Comment:
@Eilon (this is just the typescript typings for `request` below)

Reply:
Good to go.